.env.example

```diff
@@ -1,4 +1,4 @@
-DATABASE_URL=postgres://postgres:docker@dockerdns:5432/config?sslmode=disable
+DATABASE_URL=postgres://postgres:docker@localhost:5432/config?sslmode=disable
```

Comment:
changing from `dockerdns` to `localhost` as we won't have docker-in-docker, since we are moving away from Jenkins?